src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1144,8 +1144,8 @@
     <Node Name="BoundTupleCreationExpression" Base="BoundExpression">
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
-    <!-- The constructor of the underlying tuple type -->
-    <Field Name="ConstructorOpt" Type="MethodSymbol" Null="allow"/>
+    <!-- The constructor(s) of the underlying tuple type. One for short tuples. Multiple for long tuples. (largest first) -->
+    <Field Name="ConstructorChainOpt" Type="ImmutableArray&lt;MethodSymbol&gt;" Null="allow"/>
```

Comment:
The only time we need this information is when we lower this node and all constructors can be easily recomputed based on the underlying tuple type. I think we should get rid of this member.